screens/SingleDayScreen.tsx

```diff
@@ -49,6 +50,22 @@ const SingleDayScreen: React.FC<
     getForecast,
   } = useWeatherData(city, today);
 
+  const setAnimationState = (isRunning: boolean) => {
+    setIsAnimationRunning(isRunning);
+  };
+
+  const scrollHandler = useAnimatedScrollHandler((event) => {
+    scrollY.value = event.contentOffset.y;
+  });
+
+  useDerivedValue(() => {
+    if (scrollY.value > borderWhereWeStartAnimation) {
+      runOnJS(setAnimationState)(true);
+    } else {
+      runOnJS(setAnimationState)(false);
+    }
+  }, [scrollY]);
```

Comment:
`useDerivedValue` służy do tworzenia SharedValue którego wartość jest wyliczana na podstawie wartości innego SharedValue (jak zresztą sama nazwa wskazuje). Jeśli chcesz w zależności od wartości SharedValue wykonywać jakiś kod, to jest do tego [useAnimatedReaction](https://docs.swmansion.com/react-native-reanimated/docs/advanced/useAnimatedReaction/)

Reply:
@aant0ni0 Na nowym branchu czyli gdzie? Na tym `radarScreen` nie widzę żadnych nowych zmian

Reply:
Wlasnie teraz zrobilem update radarScreena, animacje byly zmienione i poprawione na nim juz duzo wczesniej, wszystko tam dziala jak nalezy